gallery.js

```diff
@@ -0,0 +1,21 @@
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+function clickListener(event) {
+  event.preventDefault();
+  const target = event.target;
```

Comment:
variable is redundant, you can use directly in next line
`const aTarget = event.target.closest('a');`